spec/controllers/admin/customers_controller_spec.rb

```diff
@@ -37,6 +37,30 @@
       locals = capture_view_locals { get :index }
       expect(locals[:show_search_bar]).to be_truthy
     end
+
+    it "sorts by id by default" do
+      customer1 = create(:customer)
+      customer2 = create(:customer)
+      customers = [customer1, customer2]
+
+      locals = capture_view_locals { get :index }
+      expect(locals[:resources].map(&:id)).to eq customers.map(&:id).sort
+    end
+
+    it "supports default sorting overrides" do
+      allow_any_instance_of(Admin::CustomersController).to(receive_messages(
```

Comment:
Is there a way we can do this without using `allow_any_instance_of`? Perhaps by creating another controller subclass inside the test?

Reply:
I replaced it with an anonymous class. Does this help clarify the test or are you thinking something else?